scripts/release/generate-changelog.js

```diff
@@ -12,9 +12,9 @@ const { modifyFile } = require('../lib/files-utils')
 
 const CHANGELOG_FILE = 'CHANGELOG.md'
 const CONTRIBUTING_FILE = 'CONTRIBUTING.md'
-const EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚗️', '♻️']
-const START_WITH_EMOJI = /^\p{Emoji_Presentation}/u
-
+const PUBLIC_EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚡️', '📝', '⚗️']
+const INTERNAL_EMOJI_PRIORITY = ['👷', '🎨', '🧪', '✅', '👌', '♻️']
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** 
- Where does 👌 come from? ‏I don't see it in [gitmoji](https://gitmoji.dev/). 
- I don't see any 🧪 in our changelog. Are we expecting to use it?

**💬 suggestion:** 
- Add 🔧 , 📄, 🔇, 🔊,📦 in the INTERNAL_EMOJI_PRIORITY

Reply:
❓ question: Could we enforce the PR title ?

Reply:
All the emojies I used comes from the [contributing.md ] (https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md)

Reply:
This emoji 🧪 was used for example for this [PR](https://github.com/DataDog/browser-sdk/pull/2821)

Reply:
🧐 hum, the contributing file is probably not up to date. Since this PR organize the changelog, I personally looked at the emojis used in the [CHANGELOG.md](https://github.com/DataDog/browser-sdk/blob/main/CHANGELOG.md)